README.md

```diff
@@ -18,4 +18,88 @@ Email or phone alerts for subscribers:
+
+### 🔭 `/astronomy-pipeline`
+Contains the files necessary to run the weekly astronomy ETL pipeline that uses data from the [Astronomy API](https://astronomyapi.com/).
+
+This data source was used to obtain information up to two weeks in advance regarding:  
+1. The relative location of celestial bodies within our solar system from different counties around the UK.
+2. Daily data on the moon phase as visible from the UK
+3. Star charts of different stellar constellations as visible from the UK.
+
+For instructions on running this pipeline please see the [Astronomy README](/astronomy-pipeline/README.md).
+
+### 🌌 `/aurorawatch-pipeline`
+Contains the files necessary to run the hourly ETL pipeline that uses data from the [AuroraWatch API](https://aurorawatch.lancs.ac.uk/).
+
+This data source was used to obtain information regarding the current aurora activity in the UK.
+
+For instructions on running this pipeline please see the [AuroraWatch README](/aurorawatch-pipeline/README.md).
+
+### 🧑‍💻 `/dashboard`
+Contains the files necessary to run the StarWatch dashboard service which is run using an AWS ECS Service. This analyses the data from all of this project's data pipelines to create insightful visualisations.
+
+This dashboard also provides users with the ability to subscribe to StarWatch for personalised astronomy SMS alerts.
+
+For instructions on running the dashboard please see the [Dashboard README](/dashboard/README.md).
+
+The [Dashboard Wireframe](/assets/starwatch_dashboard_wireframe.png) provides an overview of the visualisations that are displayed by this service.
+
+### 💾 `/database`
+Contains files that simplify the process of experimenting with and testing the RDS database linked to this repository.
+
+The [Database ERD](/assets/ERD_starwatch.png) shows the structure of this database.
+
+These files consist of:
+
+`.sh` files that can be run from the terminal using:
+```bash
+bash [filename.sh]
+```
+
+`.sql` files that can be run from the terminal using:
+```bash
+psql -U [database_username] -d [database_name] -f [path_to_file.sql]
+```
+
+For more information on these files and how to use them, please see the [Database README](/database/README.md).
+
+### 🌦️ `/quadhoral-openmeteo`
+Contains the files necessary to run the quadhoral OpenMeteo ETL pipeline that uses data from the [OpenMeteo API](https://open-meteo.com/).
+
+This pipeline extracted a range of UK weather related information that was updated every four hours to ensure that the data remained accurate and reliable.
+
+For instructions on running this pipeline please see the [Quadhoral OpenMeteo README](/quadhoral-openmeteo/README.md).
+
+### 🌅 `/weekly-openmeteo`
+Contains the files necessary to run the weekly OpenMeteo ETL pipeline that extracts sunrise and sunset data from the [OpenMeteo API](https://open-meteo.com/).
+
+For instructions on running this pipeline please see the [Weekly OpenMeteo README](/weekly-openmeteo/README.md).
```

Comment:
Good to have links to each directory

Reply:
Good links to the external APIs we used, allows for easy access for the person looking at the README.